openpype/hosts/maya/api/lib.py

```diff
@@ -293,18 +315,23 @@ def collect_animation_defs(fps=False):
     """
 
     # get scene values as defaults
-    start = cmds.playbackOptions(query=True, animationStartTime=True)
-    end = cmds.playbackOptions(query=True, animationEndTime=True)
+    frame_start = cmds.playbackOptions(query=True, minTime=True)
+    frame_end = cmds.playbackOptions(query=True, maxTime=True)
+    handle_start = cmds.playbackOptions(query=True, animationStartTime=True)
+    handle_end = cmds.playbackOptions(query=True, animationEndTime=True)
+
+    handle_start = frame_start - handle_start
+    handle_end = handle_end - frame_end
```

Comment:
Handle start and handle end appear unused? Should these still be passed to the defaults for `handleStart` and `handleEnd`?

Reply:
Yup, done.

Reply:
Is this commit still missing? :)